README.md

```diff
@@ -1,17 +1,35 @@
 # `styled-components.com`
 
-The `styled-components` website. Built with Next.js, Preact and (of course) styled-components!
+The `styled-components` website is made for documentation, showcasing, and enabling users with knowledge of the Styled Components library. 
+Built with Next.js, Preact and (of course) styled-components!
+> Styled Components is a feature-packed library for adding CSS and JavaScript styling to web applications with ease. 
 
-## Contributing
+## Table of Contents
 
-Thank you so much for wanting to contribute to the website! We could really use a hand at making it the best it can be, see [the issues](https://github.com/styled-components/styled-components-website/issues) for some ideas on what to do.
+* [Contributing Guideline](#contributing-guideline)
+* [Building A Development Server](#building-a-development-server)
+* [Folder Structure](#folder-structure)
+
+## Contributing Guideline
+
+Thank you so much for wanting to contribute to the website! We could really use a hand at making it the best it can be, see [the issues](https://github.com/styled-components/styled-components-website/issues) for some ideas on what to do. If you are new to contributing, you can find issues marked as 'good first issue' [here](https://github.com/styled-components/styled-components-website/contribute).
 
 If you've never worked with the technologies used in this repo, here are some links that may help:
 
 - [**Learn Next.js**](https://nextjs.org/learn)
 - [Next.js documentation](https://github.com/vercel/next.js)
+- [Markdown/MDX with Next.js](https://nextjs.org/blog/markdown)
 - [styled-components documentation](https://styled-components.com) (this very website!)
 
+The code in this repository uses the prettier-extension for formatting. You can find the configuration file .prettierrc in main.
+_See [prettier.io](https://prettier.io/) for more information_.
+
+Please follow the standard Github contributing guideline [found here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) when making a contribution. 
+
+## Building a Development Server
```

Comment:
Not sure this change is needed, there's a line under "Running locally" that already has this info:

> Note: This requires Node.js and yarn to be set up locally, see [nodejs.org](https://nodejs.org) for more information.